src/components/CharactersPage/CharactersTable/CharactersTable.js

```diff
@@ -86,11 +95,32 @@ const CharactersTable = ({ onRowClick }) => {
     },
   ]
 
+  if (error) console.error(error.message)
+  if (loading) return <div>loading...</div>
```

Comment:
Your app is going to crash if an error happens during data fetching, what can you do to stop that?